packages/quantic/playwright/page-object/searchObjectWithNotificationsTriggers.ts

```diff
@@ -0,0 +1,30 @@
+import {Page} from '@playwright/test';
+import {SearchObject} from './searchObject';
+
+export class SearchObjectWithNotifyTrigger extends SearchObject {
```

Comment:
Je commence a me dire qu'ont pourrais pas tout mettre les mock methods dans le searchObjects de base? Il vas avoir beacoup de searchObjectWithXX messemble hehe

Reply:
good point indeed! seems like something we can improve here!

Reply:
Yea we'll see how it evolves, but at some point we'll need to refactor for sure